resources/lang/en/notifications.php

```diff
@@ -3,7 +3,7 @@
 return [
     'campaign'          => [
         'application'           => [
-            'approved'  => 'Your application to the :campaign campaign has been approved.',
+            'approved'  => 'Your application to the :campaign campaign has been approved. Message provided: :reason',
```

Comment:
I know the rejection shows an empty reason, which isn't great UX. Can we have two strings, one with and one without the reason, and the notification is saved with the correct one? Thanks :)